src/main/java/graphql/language/AstPrinter.java

```diff
@@ -558,18 +598,12 @@ public static String printAst(Node node) {
      */
     public static String printAstCompact(Node node) {
         StringWriter sw = new StringWriter();
-        printAst(sw, node);
-        return sw.toString().replaceAll("\\s+", " ").trim();
+        printImpl(sw, node, true);
```

Comment:
Should we mention, on the Javadocs of `printAstCompact`, that comments will be suppressed?

Reply:
updated